### ISSUE TITLE
marine / terrestrial / biome field issues

### ISSUE THREAD

**Issue by peggynewman** [AtlasOfLivingAustralia/la-pipelines#464]
Feedback from OBIS - clearly there's a problem with this humpback whale record in the ocean:
https://biocache.ala.org.au/occurrences/71996a3f-60bc-4f34-b1cb-63f615503610

field | value
-- | --
Terrestrial | true
Biome | TERRESTRIAL
Marine | false

It was agreed that we would implement `biome` using the GADM spatial layers ([Confluence](https://confluence.csiro.au/display/CIU/Coordinate_habitat_mismatch+assertion%2C+species_habitats+field+and+biome+field+analysis)). Did `Marine` and `Terrestrial` come from GBIF? Should we plan to deprecate `Biome` in favour of the `Marine` and `Terrestrial` fields to avoid conflicting values?

**Comment by djtfmartin** [comment]
Yes its accuracy issue, but perhaps not as bad on second glance when you zoom right in.

<img width="545" alt="Screen Shot 2021-07-01 at 11 33 39 am" src="https://user-images.githubusercontent.com/444897/124110553-3ffa0e80-da60-11eb-84ce-5730baa15a35.png">

We are only using the Bitmap lookups for `biome`, we could also use shapefile intersects as well.

**Comment by djtfmartin** [comment]
@peggynewman 
The biocache-service is setting `Marine` and `Terrestrial` based on the `biome` value. `biome` is the only computed field in pipelines -  `Marine` and `Terrestrial` don't exist in the AVRO.

**Comment by javier-molina** [comment]
Seems reasonable to me.

> 
> We are only using the Bitmap lookups for `biome`, we could also use shapefile intersects as well.

**Comment by peggynewman** [comment]
This has been raised with us again. I'll take it over to the other repo and refer back to here.
